src/redux/categories/tests/categories.reducer.test.js

```diff
@@ -0,0 +1,36 @@
+import categoriesReducer from '../categories.reducer';
+import { SET_CATEGORIES } from '../categories.types';
+
+describe('categories.reducer.test', () => {
```

Comment:
In names of test cases, please, use capitalized plain words without dots

Reply:
Renamed